x-pack/plugins/secops/public/components/timeline/body/renderers/suricata_row_renderer.tsx

```diff
@@ -144,22 +160,41 @@ export const suricataRowRenderer: RowRenderer = {
               {signature}
             </EuiButton>
             <Details>
-              <LabelValuePairContainer>
-                <Label>{i18n.SOURCE}</Label>
-                <ValuesContainer>
-                  <DraggableValue data={data} fieldName={'source.ip'} />
-                  {':'}
-                  <DraggableValue data={data} fieldName={'source.port'} />
-                </ValuesContainer>
-              </LabelValuePairContainer>
-              <LabelValuePairContainer>
-                <Label>{i18n.DESTINATION}</Label>
-                <ValuesContainer>
-                  <DraggableValue data={data} fieldName={'destination.ip'} />
-                  {':'}
-                  <DraggableValue data={data} fieldName={'destination.port'} />
-                </ValuesContainer>
-              </LabelValuePairContainer>
+              {fieldExists({ data, fieldName: 'source.ip' }) ? (
+                <LabelValuePairContainer>
+                  <Label>{i18n.SOURCE}</Label>
+                  <ValuesContainer>
+                    <DraggableValue
+                      data={data}
+                      data-test-subj="source-ip-and-port"
+                      fieldName={'source.ip'}
+                    />
+                    {fieldExists({ data, fieldName: 'source.ip' }) &&
```

Comment:
Is this a redudant check of `source.ip`? Aren't you already checking it before you get to this section.